src/__tests__/parseErrors.test.ts

```diff
@@ -0,0 +1,63 @@
+import { ErrorList } from '../types/ErrorList'
+import { expect, test } from '@jest/globals'
+import { parseError } from '../helpers/parseError'
+
+const exampleErrors = {
```

Comment:
I can put it in another file outside of `__tests__`

Reply:
Put it into `utils` or something